### PR TITLE
Made various fixes

### PR DESCRIPTION
**Pull request by rift137**

### CODE REVIEW

File: app/controllers/actors_controller.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def create
         flash[:error] = "Person was not found."
       else
         flash[:info] = "#{person_record_for_actor.name} needs to log into this site before he/she can be given access to this profile."
-      end
```
Comment (halloffame) [inline app/controllers/actors_controller.rb]: You are missing and `end` here.

File: app/controllers/organizations_controller.rb [hunk 2]
```diff
@@ -48,10 +48,9 @@ def new_address
   end
 
   def new_organization_from_params
+    @organization = Organization.new
```
Comment (halloffame) [inline app/controllers/organizations_controller.rb]: Can you put a comment as to why you are doing this differently?

File: app/controllers/actors_controller.rb [hunk 3]
```diff
@@ -5,10 +5,10 @@ class ActorsController < ApplicationController
 
   def create
     # First find the actor's person record from the given person id
-    person_record_for_actor = Person.find(params[:_person_id])
-
-    # Find the user record for that person, since it is the user we need to authenticate
-    actor = User.where(person_id: person_record_for_actor.id).first
+    actor = if person_record_for_actor = Person.find(params[:_person_id])
+      # Find the user record for that person, since it is the user we need to authenticate
+      User.where(person_id: person_record_for_actor.id).first
```
Comment (halloffame) [inline app/controllers/actors_controller.rb]: Would it be clearer if you just did...

```ruby
if person_record_for_actor = ...
  actor = User.where...
end
```

File: app/controllers/actors_controller.rb (hunk 1 above)
Comment (rift137) [inline app/controllers/actors_controller.rb]: This is actually fine

**Pull request merged**